src/cargo/core/compiler/context/mod.rs

```diff
@@ -357,38 +349,35 @@ impl<'a, 'cfg> Context<'a, 'cfg> {
 
     /// For a package, return all targets which are registered as dependencies
     /// for that package.
+    /// NOTE: This is deprecated, use `unit_deps` instead.
```

Comment:
Hah! I had tried.  Almost every call assumed it had ownership, so they essentially became what this function is.  I'd like to try to clean this up in a follow-up, since it might require a lot of changes scattered throughout, and I figured this diff is already quite large.